examples/node_tree.py

```diff
@@ -0,0 +1,38 @@
+from napari.qt import get_app, run
```

Comment:
I like the dev folder idea,  there is indeed nothing napari in here.  get_app is required because TreeView needs an app before instantiation (we don't do the auto app creation anywhere but viewer.window).  I'll change it to use the qt app creation directly to make that clearer, after moving to dev folder

Reply:
fixed, moved to `examples/dev`, and provided more explanation in the module docstring.